_posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "2018-12-02-Reading: 7 reasons why smart, hardworking people don't become successful"
+date:   2018-12-02 19:31:07 +0800
+categories: readingnote
+---
+
+[Video Link](2018-12-02-Reading: https://www.youtube.com/watch?v=cqINcpbcZqs)
+
+#### Summary:
+1、You don't reach out to new people.
+2、You are averse to change.
+3、You're not willing to take risks.
+4、You believe you deserve success based on credentials.
+5、You constantly go after whatever's exciting at the moment.
+6、You can't commit to a decision.
+7、You don't believe in yourself.
+
+#### Reflection:
+The video is really refreshing and I think there are really some points I need to improve.
+First, make new friends especially from different area, have different life experience or have different ideas. Start by say hi to strangers, welcome different ideas, inspects, perspective etc.
```

Comment:
First, make new friends```,``` especially from different ```areas```, ```with``` different ```experiences``` or different ideas. Start by ```saying``` hi to strangers, ```welcoming``` different ideas, ```inspirations``` (not sure what you mean by inspects, maybe inspirations?), and perspectives.